common/c/targets/esp8266/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+#
+#  Copyright 2014-2016 CyberVision, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+cmake_minimum_required(VERSION 2.8)
+
+if(DEFINED $ENV{ESPRESSIF_HOME})
+    set(ESPRESSIF_HOME $ENV{ESPRESSIF_HOME})
+else()
+    set(ESPRESSIF_HOME /opt/Espressif)
+endif()
+
+set(ESP_SDK_HOME ${ESPRESSIF_HOME}/esp-rtos-sdk)
+set(ESP_TOOLCHAIN_LIBS_HOME ${ESPRESSIF_HOME}/crosstool-NG/builds/xtensa-lx106-elf/lib/gcc/xtensa-lx106-elf/4.8.2)
```

Comment:
Can you change this path or make it configurable? I use properly installed toolchain, so I don't have `crosstool-NG/builds/xtensa-lx106-elf/lib/gcc` path. Check `toolchains/esp8266.cmake` in the SDK.